pom.xml

```diff
@@ -273,6 +267,12 @@ under the License.
       <version>4.13.2</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
can we stick to pure JUnit 4.x only? Test library proliferation is simply confusing